### DISCUSSION TITLE
Ajuda

### DISCUSSION THREAD

**Discussion by Josimar** [MarceloPCF/COIR#1] [General]
Opá, boa noite, eu encontrei o seu repositório pelo curso do Jimmy, gostaria de ajudar com o projeto, também sou desenvolvedor.

No meu caso tenho conta na BTG, não está funcionando. Tem uma lista de Bugs e o que precisa ser feito?

**Comment by MarceloPCF** [comment]
Olá Josimar. Boa noite!
Você tem o código do erro? Está funcionando com minhas notas da Necton, que agora é BTG. Acredito que as notas tenham o mesmo layout, por ser da mesma corretora, ao menos deveriam ter.

Enfim, talvez o programa não identificou a corretora corretamente e acabou tentando rodar com um perfil genérico, acho que vou tirar essa opção do script.

Estou usando o tabula para identificar o nome da corretora, se o layout da nota de corretagem é diferente o sistema se perde e não consegue identificar o nome da corretora assertivamente. Isso é algo que estou pensando em mudar também, de forma que a cada exportação o usuário deve selecionar o nome da corretora.

A minha dificuldade é testar o sistema com volume maior e mais variado de dados (corretora), porque desenvolvi apenas com minhas notas de corretagem para teste. E caso você tenha analisado o código, para cada corretora há uma implementação diferente.

Se você puder me envia a sua nota para que eu possa testar e fazer os ajustes necessários, pode mandar por e-mail mesmo.

Obrigado pelo contato e pelo interesse em ajudar, vamos inicialmente deixar suas notas rodando corretamente para que vc possa vê o sistema todo funcionando.

**Comment by MarceloPCF** [comment]
A minha nota de corretagem do BTG tem essa formatação:

![Necton](https://github.com/MarceloPCF/COIR/assets/84984909/f6b31b94-c5e8-49de-997e-0c45a39230c1)

**Reply by Josimar** [reply]
Eu acredito que não seja isso, abri um ISSUE com o erro, vou olhar melhor o código hoje, fiz um commit .gitignore só para facilitar o trabalho.

Vou te atualizando.

**Comment by MarceloPCF** [comment]
O sistema conseguiu extrair os dados de todas as suas notas de corretagens corretamente?
